packages/webapp/src/pages/_app.tsx

```diff
@@ -24,6 +25,8 @@ Router.events.on("routeChangeError", () => NProgress.done());
 dayjs.extend(localizedFormat.default);
 dayjs.extend(relativeTime.default);
 
+Modal.setAppElement("#__next");
```

Comment:
curious question: why `#__next`? is it arbitrary or is it a magic keyword for next or react-modal etc?

Reply:
`Modal.setAppElement()` helps with a11y, and is intended to target the DOM element representing the app mounting point. Next tags that in the DOM with the id `__next`.